src/components/careers/Positions/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+
+import { Card } from './Card'
+import ContentWrapper from '../../Layout/ContentWrapper'
+
+const JOBS_API_URL = 'https://boards-api.greenhouse.io/v1/boards/gnosis/jobs'
+
+const SectionTitle = styled.h2`
+  font-size: 45px;
+  font-weight: bolder;
+  text-align: center;
+  margin-bottom: 56px;
+`
+
+type JobMeta = {
+  id: number
+  name: string
+  value: unknown
+  value_type: string
+}
+
+type Job = {
+  absolute_url: string
+  data_compliance: unknown[]
+  id: number
+  internal_job_id: number
+  location: {
+    name: string
+  }
+  metadata: JobMeta[]
+  requisition_id: unknown
+  title: string
+  updated_at: string
+}
+
+type Jobs = {
+  jobs: Job[]
+  meta: {
+    total: number
+  }
+}
+
+export const Positions = () => {
+  const [positions, setPositions] = useState<Jobs>()
+
+  useEffect(() => {
+    let isCurrent = true
+
+    const getOpenPositions = async () => {
+      try {
+        const data = await fetch(JOBS_API_URL)
+        const jobs: Jobs = await data.json()
+
+        if (isCurrent) {
+          setPositions(jobs)
+        }
+      } catch (e) {
+        console.error(e)
+      }
+    }
+
+    getOpenPositions()
+
+    return () => {
+      isCurrent = false
+    }
+  }, [])
```

Comment:
I would extract this into a separate hook (could be in the same file).
You might also consider copying `useAsync` into this project.